sdks/python/apache_beam/coders/standard_coders_test.py

```diff
@@ -133,11 +171,17 @@ def parse_coder(self, spec):
                      for c in spec.get('components', ())]
     context.coders.put_proto(coder_id, beam_runner_api_pb2.Coder(
         spec=beam_runner_api_pb2.FunctionSpec(
-            urn=spec['urn'], payload=spec.get('payload')),
+            urn=spec['urn'], payload=spec.get('payload', '').encode()),
```

Comment:
Why is the encode now needed? Is this a Python 3 change?

Reply:
Yeah this was just a fix I slipped in to get the test working in Python 3 and forgot about..

It should probably specify an encoding. Do you think it should be `latin1` like we do with the [expected encoded value](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/coders/standard_coders_test.py#L115)?

Reply:
Yep. Latin1 is the identity on the bottom 255 code bytes of unicode (and rejects the rest).

Reply:
Done